pages/docs/server-managers/others.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Other Server Managers
+---
+
+## Current stance on other server managers
+
+At this time, FOSSBilling only supports CPW, HestiaCP and WHM/cPanel, as server managers. 
+
+While we have received requests for support of other server managers, we are currently focusing on improving the stability and usability of FOSSBilling before adding support for additional server managers.
+
+We appreciate your interest in FOSSBilling and welcome contributions from the community. 
+If you are interested in contributing to FOSSBilling, check out visit the [contribution handbook](https://fossbilling.org/docs/contribution-handbook/guides/creating-a-server-manager) for more information.
+
+## Proxmox VE Support
+
+There is a Proxmox VE module available for FOSSBilling. 
+While Version 0.0.5 is largely not functional **and should not be used in production**, development is ongoing and we hope to have a working version available soon.
+Current progress can be tracked on the Modules [Github Repo](https://github.com/FOSSBilling/Proxmox). 
+
+## Often requested server managers
+
+Please see [this github Issue](https://github.com/FOSSBilling/FOSSBilling/issues/1362) for a list of server managers that have been requested by the community.
+
+## Special Mentions
+
+### Pterodactyl
+
+Pterodactyl is a game server manager that is often requested by the community.
+Currently, there is no active development on a Pterodactyl module for FOSSBilling from the FOSSBilling Team.
+
+However, there are community members who have been working on a Pterodactyl module for FOSSBilling. 
+See [here](https://github.com/timmy-time/FOSSBilling-Pterodactyl/) and [here](https://github.com/fabpot-del/Pterodactyl-FOSSBilling). 
```

Comment:
Don't list either of these. One hasn't been updated in 9 months and is just a clone of one of our modules (seemingly without any real support for Pterodactyl) and the other is written by someone who didn't even know how to write a FOSSBilling module (or even use PHP I believe).

In both cases, neither of them will even remotely function with FOSSBilling 0.5.X+, aren't being updated, and don't work.

But even outside of that, the wording on this page suggests that we *might* make a Pterodactyl module rather than the official stance that we currently have no plans to provide any support for it